test/statistical_tools/test_multi_sample_split.py

```diff
@@ -5,7 +5,10 @@
 import numpy as np
 from numpy.testing import assert_almost_equal, assert_equal
 
-from hidimstat.multi_sample_split import aggregate_medians, aggregate_quantiles
+from hidimstat.statistical_tools.multi_sample_split import (
+    aggregate_medians,
+    aggregate_quantiles,
+)
```

Comment:
I still think that the `aggregate_medians`  should go, since it is a (trivial) special case of `aggregate_quantiles`

Reply:
See the discussion on this topic in the PR #156:
https://github.com/mind-inria/hidimstat/pull/156#discussion_r1990140374

I don't think that this is the right PR for addressing this topic.

Reply:
What do you think?